code/game/mecha/mecha_control_console.dm

```diff
@@ -1,3 +1,6 @@
+//Note: This console is deprecated.

```

Comment:
the diff for this whole file looks like a mess because i seperated off the last half of it and changed the proc paths to fit our developer etiquette. Not much of consequence is done to this thing